pom.xml

```diff
@@ -821,12 +821,16 @@
                                         <groupId>org.testcontainers</groupId>
                                         <groupId>org.junit.jupiter</groupId>
                                         <groupId>org.junit.platform</groupId>
+                                        <!-- https://issues.redhat.com/browse/QUARKUS-1164 -->
+                                        <groupId>com.oracle.database.jdbc</groupId>
                                     </excludeGroupIds>
-                                    <!-- exclude dependencies specifying their keys
                                     <excludeKeys>
-                                        <key>org.testcontainers:db2</key>
+                                        <!-- https://issues.redhat.com/browse/QUARKUS-1304 -->
+                                        <key>com.microsoft.sqlserver:mssql-jdbc</key>
+                                        <key>com.microsoft.azure:adal4j</key>
+                                        <key>com.nimbusds:oauth2-oidc-sdk</key>
+                                        <key>com.nimbusds:content-type</key>
```

Comment:
I think just `com.microsoft.*` artifacts should be enough, the `nimbusds` should be automatically excluded as transitive dependencies.

Reply:
ah, I can just keep `com.microsoft.sqlserver:mssql-jdbc`, the rest is dependency of it

Reply:
adal4j is explicitly defined in extensions/jdbc/jdbc-mssql/runtime/pom.xml, will have to keep it there too.